src/System.Net.Sockets/src/System/Net/Sockets/_ReceiveMessageOverlappedAsyncResult.Windows.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I'm not entirely sure why _Overlapped_ have been split for *NIX. The concept of overlapped operations is particular to WinSock and can't really be translated to Unix. 

I would rather prefer to have these files unchanged and new, closer to native patterns from *NIX used for clarity.